scripts/tests/test_validate_docstrings.py

```diff
@@ -1,8 +1,10 @@
-import os
-import sys
-
-import numpy as np
 import pytest
+import string
+import random
```

Comment:
standard library imports should go before third-party module imports

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Indeed, they should!  Done.